app/src/main/java/com/hapramp/preferences/HaprampPreferenceManager.java

```diff
@@ -207,4 +199,13 @@ public void saveCurrentUserFollowings(ArrayList<String> follower) {
     return preferences.getStringSet("followings", null);
   }
 
+  public long nextPostCreationAllowedAt() {
+    return preferences.getLong("next_post_creation_allowed_at", 0);
+  }
```

Comment:
How will this behave if I create a post on some other platform and immediately try to create the post on the Android app?